### ISSUE TITLE
AC coupling issue: C14 and R24 are missing from BOM and PCB images

### ISSUE THREAD

**Issue by metapulp** [spenceraxani/Desktop-Muon-Detector#1]
I am getting ready to have Elecrow print my boards and assemble some components. While verifying the Excel BOM I only see one 10uF capacitor and two 100k resistors listed, but in the circuit layout there are 2 10uF capacitors and three 100k resistors. The quantity anomalies are found in the AC coupling in the circuit layout, but In the PCB renderings I do not see the AC coupling drawn out, nor do I see C14 (10 uF C) or R24 (100k R). I do see R9.

Did C14 and R24 get removed from the PCB design? Or are they still there somewhere, requiring the BOM to be updated to reflect the quantities?
![accouplingissue page 2](https://user-images.githubusercontent.com/15957038/33531603-48822168-d85d-11e7-8721-eb99000ebdb1.jpg)

**Comment by spenceraxani** [comment]
Hi,

The AC coupling is a new circuit that was recently added but is not yet committed to a GitHub project. Your comment seems to be attached to the oldest version of the project, but the schematic you show below is from a brand new iteration of the detector. Out of curiosity, where did you get the schematic below? GitHub doesn’t appear to attach names to comments.

This addition simply attempts to remove the DC offset (~100mV) from the peak detector circuit. 

Thanks,

Spencer

**Comment by metapulp** [comment]
The schematic is named "circuit.png" but I'm not sure which folder I downloaded it out of. I now see that the instructions pdf does not feature the AC coupling, and I also see that the Github download links on the main cosmic watch site take me to the two different project iterations. The link under cosmic watch downloads took me here.

So, did the Gerber files change to reflect the addition of the AC coupling? Does it make a big difference in quality of detection? I'm mainly interested in photon detection with the SiPMs and have been wanting to connect them to arduino. I am concerned about noisy signals. Thanks, Salem
<img width="1268" alt="circuit" src="https://user-images.githubusercontent.com/15957038/33584404-34465b94-d92d-11e7-8949-f9eabeeb424d.png">

**Comment by spenceraxani** [comment]
Hi,

Thanks for noting this. We’ll try and find the discrepancy and update it.

The Gerber that we currently provide do not have the AC coupling. It will not make a great difference in the quality of detection. The AC coupling simply improves the dynamic range of the circuit. 

What are you interested in measuring with the circuit? I might be able to give you some tips. Are you looking to measure single photons? The signal quality seems to be pretty good, but difficult to quantify with our resources. We appear to be lower than ~mV though.

Thanks,

Spencer

**Comment by metapulp** [comment]
Hi, Spencer,

I'm aiming to detect ultraweak bioluminescence. I'm not trying to quantify as yet. The closest I've seen anyone else come was with using a stereomicroscope and an electron multiplier CCD camera from Hamamatsu. That study was 8 years ago. If I know I am detecting the biophotons, I can go from there.

Thanks for this project. The closest I've seen to the muon detector was a scintillator project on Instructables, which was rather hard to follow. I did build a low light detector with arduino and a bread board, but I'm going to the next level with the SiPM. If you can be of any help, I'd really appreciate it. - Salem
![ultraweakbiophotondetector](https://user-images.githubusercontent.com/15957038/33638935-cb0f2e06-d9f6-11e7-913f-91be2a3dbc1c.png)

**Comment by spenceraxani** [comment]
Hi Salem,

Sounds like a neat idea. I do have a few  comments that you may have already thought about.

SiPMs typically have very high noise rates. For the particular SiPM that we chose for this project, the dark noise rate is about 1.6MHz. This is fine for us since our signal is typically a few dozen photons over a few nanoseconds, therefore the pulse really stands out on top of the dark noise. If your signal is spread out (say, less than a few photons per microsecond), it will be difficult to pull it out from the dark noise.

SiPMs also have a limited response to the photon frequency. The particular SiPMs that we use are most sensitive to roughly 420nm light, but will be able to detect photons over a range from roughly 325nm - 700nm (http://sensl.com/downloads/ds/DS-MicroCseries.pdf <http://sensl.com/downloads/ds/DS-MicroCseries.pdf>). 

Finally, our circuit is limited in measurement speed. Using the most up-to-date code/circuit we are able to accurately measure rates as high as 400Hz. This means that the circuit will not be able to identify single photons. The circuit was designed to be able to measure pulses that are greater than a few dozen photons, that arrive at less they about 0.5kHz.

Hope this give you some useful information on whether or not our setup would be appropriate for your measurement.

Thanks,

Spencer

**Comment by metapulp** [comment]
Hi, Spencer, Thank you for your feedback. It isn't that I'm trying to detect a single photon. I know I have a light source, but I don't know whether I am able to detect the light with an SiPM, so it is worth investing in one to try. I am working my way up from a simple low light detector. I built this one earlier this year and commented on it: http://www.instructables.com/id/Highly-sensitive-Arduino-light-sensor/
